CONTRIBUTING.md

```diff
@@ -111,6 +111,32 @@ The scripts below are used to build, test, and lint within the project.
   - Build/Test: `yarn build`
   - Linting (auto-fix): `yarn lint:fix`
 
+### Adding Plugins and Memory Connectors
+
+When considering contributions to plugins and memory connectors for Semantic
+Kernel, please note the following guidelines:
+
+#### Plugins
+
+We appreciate your interest in extending Semantic Kernel's functionality through
+plugins. However, we want to clarify our approach to hosting plugins within our
+GitHub repository. To maintain a clean and manageable codebase, we will not be
+hosting plugins directly in the Semantic Kernel GitHub repository.
+Instead, we encourage contributors to host their plugin code in separate
+repositories under their own GitHub accounts or organization. You can then
+provide a link to your plugin repository in the relevant discussions, issues,
+or documentation within the Semantic Kernel repository. This approach ensures
+that each plugin can be maintained independently and allows for easier tracking
+of updates and issues specific to each plugin.
```

Comment:
nit: a lot of "repository" repetitions

Reply:
![image](https://github.com/microsoft/semantic-kernel/assets/371009/c832866d-97fc-4eb1-b55c-093e70d3f424)